### PR TITLE
`README.md`: Add instructions for building ARM and ARM64.

### LINKED ISSUE #1211: README.md: Consider explaining how to build ARM and ARM64
We merged #873, removing an incomplete explanation of how to build the STL for ARM and ARM64. We should consider adding a complete explanation, which would involve selecting ARM and ARM64 support in the VS installer. #3730 added CMake Presets, centralizing everything else.

### PR DESCRIPTION
**Pull request by fsb4000**
Fixes #1211 
ARM
ARM64
arm64ec

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -180,6 +188,30 @@ To build the x64 target (recommended):
 3. `cmake --preset x64`
 4. `cmake --build --preset x64`
 
+To build the ARM target:
+
+1. Open an "x64 Native Tools Command Prompt for VS 2022 Preview".
+2. Change directories to the previously cloned `STL` directory.
+3. `"C:\Program Files\Microsoft Visual Studio\2022\Preview\VC\Auxiliary\Build\vcvarsall.bat" amd64_arm`
```
Comment (cpplearner) [inline README.md:195]: This assumes that VS 2022 is installed under `C:\Program Files`, which is not always the case.

Comment (fsb4000) [reply README.md]: изображение

File: README.md [hunk 2]
```diff
@@ -180,6 +188,30 @@ To build the x64 target (recommended):
 3. `cmake --preset x64`
 4. `cmake --build --preset x64`
 
+To build the ARM target:
+
+1. Open an "x64 Native Tools Command Prompt for VS 2022 Preview".
+2. Change directories to the previously cloned `STL` directory.
+3. `"%VSINSTALLDIR%VC\Auxiliary\Build\vcvarsall.bat" amd64_arm`
```
Comment (StephanTLavavej) [inline README.md:195]: This results in a horrible mess for the INCLUDE, PATH, etc. environment variables. It works, but I think it's easier to depict the default path and tell people to adjust accordingly if they've customized the location.

Comment (StephanTLavavej) [inline README.md:195]: vcvarsall.bat understands `x64_MEOW` as a synonym, and we try to prefer the vendor-neutral and less-confusing x64 whenever possible.

**Review by StephanTLavavej — APPROVED** [review]

**Comment by StephanTLavavej** [comment]
I'm mirroring this to the MSVC-internal repo - please notify me if any further changes are pushed.

**Comment by StephanTLavavej** [comment]
:scroll: :muscle: :mechanical_arm:
===

**Pull request merged**